internal/pacseek/setup.go

```diff
@@ -243,6 +244,15 @@ func (ps *UI) setupKeyBindings() {
 			}
 		}
 
+		// CTRL+O - Show URL for selected package
+		if event.Key() == tcell.KeyCtrlO {
+			if ps.selectedPackage == nil {
```

Comment:
This can/should be done in one if statement. f.e.
```go
if event.Key() == tcell.KeyCtrlO && ps.selectedPackage != nil {
	exec.Command("xdg-open", ps.selectedPackage.URL).Run();
	return nil
}
```